routes/web.php

```diff
@@ -142,6 +142,8 @@
                 Route::get('/remove-pdf/{id}', 'TrainingDash@removePublicPdf');
             });
         });
+        Route::get('/statistics', 'TrainingDash@statistics')->middleware('role:atm|datm|ta|wm');
+        Route::post('/statistics/graph', 'TrainingDash@generateGraphs')->middleware('role:atm|datm|ta|wm');
```

Comment:
Can use the following to always match senior staff:
```php
->middleware('permission:snrStaff')
```

Reply:
Routes updated - thanks